src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -58,6 +58,28 @@ function warnIfValueIsNull(props) {
   }
 }
 
+function warnIfDuplicateValues(inst) {
```

Comment:
We're trying to put new warnings within devtools.  For an example that you can mimic, see: https://github.com/facebook/react/pull/7040